evap/staff/tests/test_views.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+import xlrd
+from django.conf import settings
+from django.contrib.auth.models import Group
+from django.test.utils import override_settings
+from django_webtest import WebTest
+from model_mommy import mommy
+
+from evap.evaluation.models import Semester, UserProfile, Course, CourseType
+from evap.evaluation.tests.test_utils import lastform, ViewTest
+
+
+class TestSemesterExportView(ViewTest):
+    url = '/staff/semester/1/export'
+    users = ['staff']
+
+    def setUp(self):
+        self.semester = mommy.make(Semester, name='fuk', name_en='fuk', name_de='fuk')
```

Comment:
can you remove the names here and for the coursetype below and let mommy create the names?